src/list.json

```diff
@@ -243,6 +243,13 @@
         ],
         "description": "Contribute to ServiceNow to receive a free tshirt",
         "link": "https://developer.servicenow.com/blog.do?p=/post/hacktoberfest-2022/"
+      },
+      {
```

Comment:
The Code need to be Re-styled Please paste this command in the Repo Directory.
`git checkout <your branch>`
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2339283/patch | git am`
`git push`

Reply:
@thefool76 I have made the changes
Please check